### ISSUE TITLE
Possible performance regression in v1.9

### ISSUE THREAD

**Issue by klaemo** [jscs-dev/node-jscs#878]
Hey there,
I've noticed that with the latest version jscs seems to run a lot slower. So I conducted some experiments to confirm:

```
$ npm i -g jscs@1.8.1
$ jscs --version
1.8.1
$ time jscs index.js
No code style errors found.

real	0m0.346s
user	0m0.292s
sys	0m0.049s
```

```
$ npm i -g jscs@latest
$ jscs --version
1.9.0
$ time jscs index.js
No code style errors found.

real	0m2.571s
user	0m2.481s
sys	0m0.073s
```
~350ms vs 2.5s (index.js is only 1.3kb)

This is my `.jscsrc`
```
{
    "preset": "airbnb",
    "excludeFiles": ["node_modules", "public", "build", "test/browser"],
    "disallowMultipleVarDecl": "exceptUndefined",
    "requireMultipleVarDecl": null,
    "safeContextKeyword": ["self"]
}
```

Can anyone confirm this?

**Comment by mrjoelkemp** [comment]
Thanks for reporting. I felt the same with simply using the google preset yesterday. Will look into it.

**Comment by markelog** [comment]
Sounds like a duplicate of https://github.com/jscs-dev/node-jscs/pull/861, please try it with current master, if problem persist i will reopen this.

Otherwise, fix will come with 1.9.1, which should be released in couple of days, i hope that is

**Comment by klaemo** [comment]
Sorry for the duplicate issue, I must have overlooked the other one.

master fixes the issue indeed:

```
time jscs index.js
No code style errors found.

real	0m0.367s
user	0m0.309s
sys	0m0.051s
```

Thanks for the quick reply. Looking forward to 1.9.1.
